chapter04/c04_end.txt

```diff
@@ -12,10 +12,10 @@ label c04_end:
     I wonder... [Pause]  Why is he so quiet?
     Could it be that he doesn't like me? [Pause]  Or perhaps he suffers from social anxiety?
     Either way, he is really hard to read... [Pause]  That makes him mysterious.
-    As the silent atmosphere thickened , the light that had once covered me suddenly disappeared, catching my attention.
+    As the silent atmosphere thickened, the light that had once covered me suddenly disappeared, catching my attention.
     ??? "Oh my! [Pause]  It looks like someone had already beaten me here. And a young woman to boot!"
     k "?!"
-    Noticing Kenta suddenly jolting out of surprise, I looked back and forth between the two.
+    Noticing Kenta suddenly jolting up out of surprise, I looked back and forth between the two.
     c "Um... [Pause]  May I ask who you are?"
```

Comment:
This is more an editor's job, but I notice Celia doesn't give her name to Artil here.